components/docker/component.go

```diff
@@ -148,13 +148,28 @@ func (d *Manager) install() (*remote.Command, error) {
 		return nil, err
 	}
 
+	// #incident-31160: Patch ip range that docker uses to create is bridge networks
```

Comment:
I think you can just state it conflicts with other private ip usage, it's still open source code.